From 9313d3d856e74480ec583e8c8ea32adf198d8b72 Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@nxp.com>
Date: Wed, 18 Apr 2018 23:02:36 +0300
Subject: [PATCH 198/741] sdk_fman: disable timer code in arm code

commit 2411c7ea5263bf8b6f931977031f39eac2673f55 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

The (unused) timer code is no longer compatible with the newer
kernel API. Disabling the incompatible code in the arm64 code.

Signed-off-by: Madalin Bucur <madalin.bucur@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_arm_linux.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_arm_linux.c b/drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_arm_linux.c
index dd3e376e96af..ea59e413ce80 100644
--- a/drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_arm_linux.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_arm_linux.c
@@ -607,7 +607,7 @@ uint32_t XX_CurrentTime(void)
     return (jiffies*1000)/HZ;
 }
 
-
+#if 0
 t_Handle XX_CreateTimer(void)
 {
     struct timer_list *p_Timer = (struct timer_list *)XX_Malloc(sizeof(struct timer_list));
@@ -684,6 +684,7 @@ int XX_TimerIsActive(t_Handle h_Timer)
 {
   return timer_pending((struct timer_list *)h_Timer);
 }
+#endif
 
 uint32_t XX_Sleep(uint32_t msecs)
 {
-- 
2.17.1

